.config/forest.dic

```diff
@@ -31,15 +31,16 @@ tipset/SM
 tipsetkey/S
 tuple
 validator/S
-serializing
-deserializing
 mempool
 sortition
+serializer/SM
+deserialize/DG
```

Comment:
Consider using `de-serialize`/ `de-serializer` in our codebase so we don't need to add those in `forest.dic`.

Reply:
Done